docs/Roadmap/Stage08.md

```diff
@@ -321,7 +321,7 @@ Your NITCbase now supports the creation of relations. With that, we have now imp
 
 ## Exercises
 
-**Q1.** In your NITCbase, run the file [s8test.txt](/roadmap_files/stage8/script.txt) to test your implementation. Place the files [products.csv](/roadmap_files/stage8/products.txt) and [stores.csv](/roadmap_files/stage8/stores.txt) in the `Files/Input_Files` directory. Place [s8test.txt](/roadmap_files/stage8/script.txt) in the `Files/Batch_Execution_Files` directory. Once you have placed the files, execute the [run](../User%20Interface%20Commands/utility.md#run-batch-execution-command) command in your NITCbase as below.
+**Q1.** In your NITCbase, run the file [s8test.txt](/roadmap_files/stage8/script.txt) to test your implementation. Place the files [s8products.csv](/roadmap_files/stage8/products.txt) and [s8stores.csv](/roadmap_files/stage8/stores.txt) in the `Files/Input_Files` directory. Place [s8test.txt](/roadmap_files/stage8/script.txt) in the `Files/Batch_Execution_Files` directory. Once you have placed the files, execute the [run](../User%20Interface%20Commands/utility.md#run-batch-execution-command) command in your NITCbase as below.
```

Comment:
The file give to run uses the name s8stores.txt s8products.txt for inserting records. Here is that section from that code: 

echo ----Populating new relations----
INSERT INTO Products VALUES FROM s8products.csv
INSERT INTO Stores VALUES FROM s8stores.csv
echo --------------------------------
echo
echo ----Selecting from new relation----
SELECT * FROM Stores INTO null WHERE owner=Gomez
echo -----------------------------------

It might be better if we change the names in the documentation or in the command file s8.txt